docs/docs/sourcing-from-drupal.md

```diff
@@ -2,7 +2,87 @@
 title: Sourcing from Drupal
 ---
 
-This is a stub. Help our community expand it.
+## Why use Drupal + Gatsby together?
+
+Using Drupal as a headless CMS with Gatsby is a great way to get an enterprise-quality CMS for free, paired with a great modern development experience and all the benefits of the JAMstack, like performance, scalability, and security.
+
+It only takes a few steps to use Gatsby with Drupal as a headless CMS (also known as decoupled Drupal).
+
+[An example Gatsby + Drupal site](https://github.com/gatsbyjs/gatsby/tree/master/examples/using-drupal)
+
+## How to implement Drupal + Gatsby
+
+### Quick start
+This guide assumes that you have a Gatsby project set up. If you need to set up a project, head to the [Quick Start guide](/docs), then come back.
+
+### gatsby-config.js
+Hooking up Gatsby to a new or existing Drupal site takes only a few steps.
+
+- Follow the `gatsby-source-drupal` [installation instructions](/packages/gatsby-source-drupal/?=drupal) to add the plugin to your Gatsby site
+
+An example of how to include the `gatsby-source-drupal` plugin in your `gatsby-config.js` file:
+
+```
+javascript:title=gatsby-config.js
```

Comment:
That needs to be on the same line as the back-ticke

Reply:
Done! :+1: